### DISCUSSION TITLE
2023-09-21

### DISCUSSION THREAD

**Discussion by woosung1223** [woowacourse-teams/2023-haru-study#544] [daily_meeting]
### 오늘의 할 일 
- 백엔드
  - 테이블 별로 쿼리 정리
  
- 프론트엔드

- 공통
  - 오전에 새로운 버전 배포

- 개인 일정
  - 룩소 근로 (4시)
  - 노아 근로 (5시)

### 내일의 할 일

**Comment by aak2075** [comment]
### 체크인

9점 - KTX 지연때문에 지각 (-1)

**Comment by jaehee329** [comment]
### 체크인
7점 - 오늘 아침은 바로 일어나기 힘들더라구요

**Comment by MoonJeWoong** [comment]
### 체크인
8점 - 데모데이 빨리 끝내고 싶네요

**Comment by woosung1223** [comment]
### 체크인
8점 - 날씨가 너무 추워서 깜짝 놀랐네요

**Comment by yeopto** [comment]
### 체크인

10점 - 시원합니다 날씨가

**Comment by woo-jk** [comment]
### 체크인
7점 - 낼 발표네요 ㅠ

**Reply by aak2075** [reply]
화이팅

**Comment by nlom0218** [comment]
### 체크인

10점 - 9시 20분에 일어나서 초초한 기분이 은근 좋네요
